src/main/java/entities/User.java

```diff
@@ -43,9 +43,7 @@ public User(String password, String accountName, Profile profile){
 
     public Profile displayProfile(){ return this.profile;}
 
-    public List<User> showBlocked(){ return this.blocked;}
-
-    public List<User> showLiked(){ return this.liked;}
+    public List<String> showLiked(){ return this.liked;}
```

Comment:
.toSeconds() method is not working sometimes, you can use getSeconds() method instead.